openapi-contracts/src/main/com/sinch/sdk/domains/verification/models/v1/start/response/StartVerificationResponseDataContentImpl.java

```diff
@@ -1,63 +1,40 @@
-/*
- * Verification | Sinch
- * Verify users with SMS, flash calls (missed calls), a regular call, or data verification.
- *
- * The version of the OpenAPI document: 2.0.1
- * Contact: support@sinch.com
- *
- * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
- * https://openapi-generator.tech
- * Do not edit the class manually.
- */
-
-package com.sinch.sdk.domains.verification.models.dto.v1;
+package com.sinch.sdk.domains.verification.models.v1.start.response;
 
 import com.fasterxml.jackson.annotation.JsonFilter;
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import com.sinch.sdk.core.models.OptionalValue;
 import java.util.Objects;
 
-/** The response contains the target URI. */
-@JsonPropertyOrder({DataInitiateVerificationResponseSeamlessDto.JSON_PROPERTY_TARGET_URI})
+@JsonPropertyOrder({StartVerificationResponseDataContentImpl.JSON_PROPERTY_TARGET_URI})
 @JsonFilter("uninitializedFilter")
 @JsonInclude(value = JsonInclude.Include.CUSTOM)
-public class DataInitiateVerificationResponseSeamlessDto {
+public class StartVerificationResponseDataContentImpl
+    implements StartVerificationResponseDataContent {
   private static final long serialVersionUID = 1L;
+
   public static final String JSON_PROPERTY_TARGET_URI = "targetUri";
-  private String targetUri;
-  private boolean targetUriDefined = false;
 
-  public DataInitiateVerificationResponseSeamlessDto() {}
+  private OptionalValue<String> targetUri;
 
-  public DataInitiateVerificationResponseSeamlessDto targetUri(String targetUri) {
-    this.targetUri = targetUri;
-    this.targetUriDefined = true;
-    return this;
-  }
+  public StartVerificationResponseDataContentImpl() {}
 
-  /**
-   * The target URI.
-   *
-   * @return targetUri
-   */
-  @JsonProperty(JSON_PROPERTY_TARGET_URI)
-  @JsonInclude(value = JsonInclude.Include.USE_DEFAULTS)
-  public String getTargetUri() {
-    return targetUri;
+  protected StartVerificationResponseDataContentImpl(OptionalValue<String> targetUri) {
+    this.targetUri = targetUri;
   }
 
   @JsonIgnore
-  public boolean getTargetUriDefined() {
-    return targetUriDefined;
+  public String getTargetUri() {
```

Comment:
Yes, the general pattern for generated files and getters is:
- there 2 getters: one public, one for internal usage
- public (documented from interface definition)  getter is in form of : `getXXX` and it return the value (or null if not set)
- internal (undocumented not declared from at interface level): getter is just in form `XXX` and return an  `OptionalValue` (not an Optional). The OptionalValue is used to check if value was present onto received payload from (for responses) or is having to be defined when sending payload from client (requests). This feature is to 
   - just sent what was set from SDK end user
   - prevent to send a NULL value if not set by SDK user. If a null value is sent, this could mean reset the value from server side. We have to prevent this.
   
So finally, why the second getter form is not public ? It could be in the future if SDK end user use case could require it; but currently to avoid to much public (and so to be maintained) feature it is not